tensorflow/compiler/xla/service/gpu/target_util.cc

```diff
@@ -46,29 +46,41 @@ struct TargetFunctionInfo {
   TargetFunctionInfo()
       : callee_name(""),
         input_types({PRIMITIVE_TYPE_INVALID}),
-        result_type(PRIMITIVE_TYPE_INVALID),
-        use_bitcast(false) {}
+        result_type(PRIMITIVE_TYPE_INVALID) {}
```

Comment:
@sumesh13 , `check_signature` is not populated in the initialization list here in the default ctor.

Reply:
I removed the default ctor. Dont think I need it.